Python/Background_Remover/main.py

```diff
@@ -0,0 +1,75 @@
+import cv2
+import numpy as np
+
+#== Parameters =======================================================================
+BLUR = 21
+CANNY_THRESH_1 = 10
+CANNY_THRESH_2 = 200
+MASK_DILATE_ITER = 10
+MASK_ERODE_ITER = 10
+
+# In BGR format
+MASK_COLOR = (0.0,0.0,0.0)
+
+
```

Comment:
Few changes in the implementation:
* Add `modularity` in the code by making OOP classes or atleast make UDFs for the same.
* This whole thing works in a data flow pipeline. So I would suggest briefly explain them in comments as well as add functions accordingly. 
* Data Pipeline means data undergoes various operations and then returned as an end result. Here it is evident that first the data is read, then edges are detected, then contours, masking, channeling etc.